lib/mini_profiler/profiler.rb

```diff
@@ -308,11 +323,13 @@ def call(env)
       end
 
       if query_string =~ /pp=env/ && !config.disable_env_dump
```

Comment:
@SamSaffron there is an existing config option for env, should I deprecate this one in favor of the new option?

Reply:
yes lets remove that in favor of the new one! 

Once done feel free to merge.